microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/v20/MicroProfileOpenApi20Test.java

```diff
@@ -0,0 +1,224 @@
+        Map<String, Object> yamlMap = (Map<String, Object>) yamlObject;
+
+        Map<String, Object> paths = (Map<String, Object>) yamlMap.get("paths");
+        Assert.assertFalse("\"paths\" property is empty", paths.isEmpty());
+
+        // 2.0 The @SchemaProperty annotation has been added to allow the properties for a schema to be defined inline
+        Map<String, Object> getDistrictByCodePath = (Map<String, Object>) paths.get("/districts/{code}");
+        Assert.assertFalse("\"/districts/{code}\" property is empty", getDistrictByCodePath.isEmpty());
+
+        Map<String, Object> getMethod = (Map<String, Object>) getDistrictByCodePath.get("get");
+        Assert.assertFalse("\"/districts/{code}\" \"get\" property is empty", getMethod.isEmpty());
+        Assert.assertEquals("\"/districts/{code}\" \"get\" property should have exactly 6 keys",
+                6, getMethod.keySet().size());
+
+        Map<String, Object> responses = (Map<String, Object>) getMethod.get("responses");
+        Assert.assertFalse("\"/districts/{code}\" \"responses\" for GET verb is empty", responses.isEmpty());
+
+        Map<String, Object> http200Response = (Map<String, Object>) responses.get("200");
+        Assert.assertFalse("\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 is empty",
+                http200Response.isEmpty());
+
+        Map<String, Object> contentAnnotation = (Map<String, Object>) http200Response.get("content");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has empty \"content\" property",
+                contentAnnotation.isEmpty());
+
+        Map<String, Object> contentTypeJson = (Map<String, Object>) contentAnnotation.get("application/json");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"content\" but empty \"application/json\" property",
+                contentTypeJson.isEmpty());
+
+        Map<String, Object> schema = (Map<String, Object>) contentTypeJson.get("schema");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"application/json\" but empty \"schema\" property",
+                schema.isEmpty());
+
+        Map<String, Object> schemaProperties = (Map<String, Object>) schema.get("properties");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"schema\" but empty \"properties\" property",
+                schemaProperties.isEmpty());
+
+        Assert.assertTrue(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"properties\" but missing \"alias\" property",
+                schemaProperties.containsKey("alias"));
+
+        Assert.assertTrue("", ((LinkedHashMap) schemaProperties.get("alias")).containsValue("Alias of district"));
```

Comment:
I was wondering whether here it should be checked for just one occurrence, which would then be de-referenced and tested. WDYT?
Anyway, not such a big deal.